utils/guidelines_xslt/odd2html/guidelines.xsl

```diff
@@ -265,7 +265,7 @@
     <xsl:template match="tei:ref" mode="guidelines">
         <xsl:choose>
             <xsl:when test="starts-with(@target,'#')">
-                <xsl:variable name="chapter.id" select="replace(@target,'#','')" as="xs:string"/>
+                <xsl:variable name="chapter.id" select="substring(@target,2)" as="xs:string"/>
```

Comment:
this is much safer than replacing any `#` alternatively a substring-after could be used

Reply:
I think with the test in the preceding line this statement is clearer than `substring-after`.